app/services/irs_attempts_api/tracker_events.rb

```diff
@@ -18,6 +18,15 @@ def account_reset_request_submitted(success:)
       )
     end
 
+    # @param [String] upload_method String of method chosen for uploading id verification
+    # A user has selected id document upload method ('mobile'/'desktop')
```

Comment:
if we expect the input to be either mobile or desktop, we could write the type like this:
```suggestion
    # @param ["mobile", "desktop"] upload_method method chosen for uploading id verification
    # A user has selected id document upload method
```

Reply:
Thanks!  Done!